### ISSUE TITLE
BUG: Message from the leads is not shown anywhere

### ISSUE THREAD

**Issue by SimoneZaza** [alkem-io/notifications#355]
**Describe the bug**
We still have the functionality to write an "update from the lead". However, this message is currently only shown in the admin section. The notification shows a button to "Have a look" but this brings the user to the (sub)space where nothing is shown. 
Quick fix for now will be to embed the message in the email. Later we can have a look at how to structurally solve this. 

<img width="1528" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNmVwQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--13ae9f01aa5f6e2838cef4d5c8be94cd891057da/image.png" alt="image.png" />

**To Reproduce**

**Expected behavior**
Add message to the notification

**Comment by hero101** [comment]
is this a high priority? is it found in production?

**Comment by SimoneZaza** [comment]
Yes, a lead (admin) used this functionality on production last week.

**Comment by bobbykolev** [comment]
As I was completing a PR in that area, I'll use the opportunity to pick this up.

**Comment by SimoneZaza** [comment]
As we're going to add it to the subspace, this notification update is less of a priority, so moving it to icebox
